pkg/reconciler/apiserversource/apiserversource.go

```diff
@@ -259,13 +259,22 @@ func (r *Reconciler) createEventTypes(ctx context.Context, src *v1alpha1.ApiServ
 		return err
 	}
 
-	missing := r.computeDiff(current, expected)
-	for _, eventType := range missing {
+	toCreate, toDelete := r.computeDiff(current, expected)
```

Comment:
I would expect a test change to be required?

Reply:
Let me add one, yes... I was going to do it in the follow up, but it makes more sense to do it now

Reply:
Done, thanks for the comment!